etc/ci_scripts/dump_artifacts.py

```diff
@@ -117,6 +117,19 @@ def _dump_docker_log(container_name: str, dir: Path) -> None:
             shell=True,
         )
 
+def dump_docker_ps(dir: Path) -> None:
+    """
+    run `docker ps` and dump to $DIR/docker_ps.log
+    """
+    destination = dir / "docker_ps.log"
+    logging.info(f"Dumping 'docker ps' to '{destination}'")
+    with open(destination, "wb") as out_stream:
+        subprocess.run(
+            f"docker ps",
+            stdout=out_stream,
+            stderr=subprocess.DEVNULL,
```

Comment:
Might be useful to capture stderr together with stdout, no?

Reply:
done